simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryViewModel.kt

```diff
@@ -59,11 +59,14 @@ class AccountListRegistryViewModel(private val locationManager: LocationManager)
       this.profilesController.accountEvents()
         .observeOn(AndroidSchedulers.mainThread())
         .subscribe(this::onAccountEvent),
-      Observable.combineLatest(queries, locationUpdates) { query, _ -> createQuery(query) }
-        .switchMapCompletable(this::executeQuery)
-        .subscribeOn(Schedulers.io())
-        .onErrorComplete()
-        .subscribe()
+      /**
+       * Here we are hiding Library Finder functionality
+       */
+//      Observable.combineLatest(queries, locationUpdates) { query, _ -> createQuery(query) }
+//        .switchMapCompletable(this::executeQuery)
+//        .subscribeOn(Schedulers.io())
+//        .onErrorComplete()
+//        .subscribe()
```

Comment:
Just want to confirm that commenting this out doesn't completely remove any library registry queries...

If you're confident that it doesn't, then this is fine.